include/swift/AST/Expr.h

```diff
@@ -2110,6 +2116,13 @@ class CollectionExpr : public Expr {
   void setElement(unsigned i, Expr *E) { getElements()[i] = E; }
   unsigned getNumElements() const { return Bits.CollectionExpr.NumSubExprs; }
 
+  const MutableArrayRef<ConfigMapPair> &getConditionalsMap() const {
+    return ConditionalsMap;
+  }
+  void setConditionalsMap(const MutableArrayRef<ConfigMapPair> &Map) {
```

Comment:
Nitpick: since ArrayRef and MutableArrayRef are just a pointer/length pair, we usually pass them around by value. It also seems like these don't need to be Mutable at all—you never mutate them after they're set.